wiki/604838-SubmissionDashboards.md

```diff
@@ -13,9 +13,31 @@ This section provides dashboards to the person who submit NLP Sandbox tools for
 - You can cancel a submission tthat has not started yet.
 - Only submissions that have successfully completed are counted toward your (or your team's) submission quota.
 - If you can't figure out why your submission failed, please [open a ticket].
+- The `SubmissionLogs` column contains information of the workflow that is ran per submission. It doens't actually contain logs generated by your submissions. If your submission is INVALID or REJECTED, you will have to request for logs.
```

Comment:
@cascadianblue was confused when he looked at the logs. He knew his submission failed via its status but could not understand that was wrong from the log. Is there a reason why we share the workflow logs with the users if they do not include tool log? Can we return include the tool output in the log for the i2b2 dataset?

Reply:
The workflow logs is something that is returned by the orchestrator which is separate from the tool logs.  The tools logs can be returned by the ELK stack and has to be requested even for the i2b2 dataset.

Reply:
@thomasyu888 Then we should remove the log column to not give false hope to participants and eliminate the confusion that they may encounter when parsing the workflow log.